src/dev/impl/DevToys/UI/Controls/CustomTextBox.xaml.cs

```diff
@@ -139,7 +139,11 @@ public CustomTextBox()
 
         private bool CanExecuteCutCommand()
         {
-            return RichEditBox != null && RichEditBox.TextDocument.Selection.Length != 0 && IsEnabled && RichEditBox.TextDocument.CanCopy();
+            return RichEditBox != null
```

Comment:
The one line became longer, so it was made into multiple lines. It can be undone with code style.